core-backend/sandbox/src/funcs.rs

```diff
@@ -804,9 +919,12 @@ where
 
     /// Fallible `gr_system_reserve_gas` syscall.
     pub fn system_reserve_gas(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
-        sys_trace!(target: "syscalls", "system_reserve_gas, args = {}", args_to_str(args));
+        let (gas, err_len_ptr) = args
+            .iter()
+            .read_2()
+            .expect("There are 2 arguments expected");
```

Comment:
please move this panics inside helper `read_*` functions. Also we use 
```rust
.unwrap_or_else(|| unreachable!("..."))
```
instead of `expect` to unify format of panics exprs inside runtime.